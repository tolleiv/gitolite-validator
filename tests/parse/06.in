# managers should be able to read any repo
repo @allbroken
    R                       =   @managers

# ...other rules for other repos...

repo foo bar
    RW+                     =   alice @teamleads
    RW                      =   dilbert @devteam
    R                       =   @managers
