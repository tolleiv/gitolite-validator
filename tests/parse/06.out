root:
  repo:
    paths:
      @allbroken
    rules:
      rule:
        permission: R
        members:
          @managers
  repo:
    paths:
      foo
      bar
    rules:
      rule:
        permission: RW+
        members:
          alice
          @teamleads
      rule:
        permission: RW
        members:
          dilbert
          @devteam
      rule:
        permission: R
        members:
          @managers
